### PR TITLE
Feature/tempest

### PR DESCRIPTION
**Pull request by magohl**
Adds support for tempest device

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -211,12 +211,57 @@
...
+          "id": "add_devices",
+          "template": "add_devices"
+        }
+      ]
     }
   ],
   "flow": {
     "triggers": [
       {
-        "id": "rain_start",
```
Comment (WillooWisp) [inline app.json:219]: Doesn't renaming the ids for flow triggers break existing flows users have configured?

File: app.json [hunk 2]
```diff
@@ -255,7 +315,31 @@
         ]
       },
       {
-        "id": "wind_below",
+        "id": "wind_above_tempest",
```
Comment (WillooWisp) [inline app.json:318]: Do we really need unique triggers for tempest? Can we use the same and filter on what kind of device as a setting instead?

Comment (magohl) [reply app.json:318]: Makes sense to use the same trigger but with a device filter. I can fix it within the next couple of days if you want. Or will you fix it?

File: app.json (hunk 1 above)
Comment (magohl) [reply app.json:219]: I am new to Homey development but that makes sense. Let keep them the id the same.

File: app.json (hunk 2 above)
Comment (WillooWisp) [reply app.json:318]: I think you should fix it and test it with tempest, then I can verify that it still works with sky

Comment (magohl) [reply app.json:318]: I made a quick test where one common trigger, with two args of type device, is used. But it did not work as expected. I get the "Error: invalid_flow_card_id". I now remember trying that earlier on as well before going for separate ones. Can you think of something, maybe in the app/onInit, that could cause this behavior?

Comment (WillooWisp) [reply app.json:318]: Take a look at, https://github.com/rogro82/com.hue.zigbee/blob/master/app.json, where device is an argument to the trigger.

Comment (WillooWisp) [reply app.json:318]: Did you understand from this sample how it can be done?

Comment (WillooWisp) [reply app.json:318]: Have you given up? 😀

Comment (WillooWisp) [reply app.json:318]: I have fixed it now.

**Comment by WillooWisp** [comment]
I have created a test version of the Homey app now with support for Tempest, https://homey.app/a/com.weatherflow.smartweatherstation/test/

File: app.json (hunk 2 above)
Comment (magohl) [reply app.json:318]: Sounds great. Will install tonight and try it out, just need some more light to power the Tempest :)

**Pull request closed without merging**